test/index.js

```diff
@@ -109,6 +110,33 @@ tester.describe("scrape-it", t => {
             cb();
         });
     });
+    t.it("closet sample", cb => {
+        scrapeIt(URL, {
+            addresses: {
+                listItem: "table tbody tr",
+                data: {
+                    address: ".address",
+                    suburb: {
+                        selector: ".city",
+                        closest: "table",
+                        convert(html) {
+                            $ = cheerio.load(html);
```

Comment:
@hjc1983 After your latest change, you'll receive the `$node` element in the second parameter of this `convert` function. Then you can simply do:

```js
convert (html, $node) {
  return $node.find(".city").text();
}
```